src/modules/integrator-guide/pages/security-model/users.adoc

```diff
@@ -6,16 +6,14 @@ include::ROOT:partial$variables.adoc[]
 :description: Learn how to create and manage your platform users
 
 A user is an entity which is allowed to interact with your {PRODUCT_NAME} instance.
-A user can be an employee who needs to interact with their, or a system user which may be used to identify external integrations or webhook requests.
-//Editor's  note: ...who needs to interact with their (what?) - it's better to overstate than understate, especially when you have big chunks of text between the possessive pronoun and the reference.
+A user can be an employee who needs to interact with your {PRODUCT_NAME}, or a system user which may be used to identify external integrations or webhook requests.
 
 A user can be assigned to a series of xref:security-model/roles.adoc[roles] which define the scope of the access for the given user.
 
 [IMPORTANT]
 ====
 For privacy reasons, personal information is masked by default.
 This can be configured by setting appropriate access control rules for either all users or a specific user.
-====
```

Comment:
I insist that it is reworked since the original version lacks clarity. The "A user can be an employee who needs to interact with your {PRODUCT_NAME}, or a system user which may be used to identify external integrations or webhook requests." sounds good